app/services/query-service.js

```diff
@@ -123,13 +129,13 @@ const runFetchAnswerQuery = async ({ query, chatHistory, summariesMode, embeddin
       requestQuery: query
     })
     return {
-      response: { answer: 'This tool cannot answer that kind of question, ask something about Defra funding instead' }
+      response: { answer: 'This tool cannot answer that kind of question, ask something about Defra funding instead', hallucinated: true }
     }
   }
 }
 
 const fetchAnswer = async (req, query, chatHistory, cacheEnabled, summariesEnabled = false) => {
-  if (cacheEnabled) {
+  if (cacheEnabled && !summariesEnabled) {
```

Comment:
Any reason why summaries shouldn't be uploaded to cache also?

Reply:
I thought because summaries index is not filled with content just yet and thus when it finds a response, it is likely to be incomplete vis-a-vis the content in the actual index search

Reply:
I've ran the data ingester completely, so summaries index should be full

Reply:
Test it out for yourself before making the call but it should be ok.